### PR TITLE
Add Get & Sets for the warningTimeSeconds method & adding more proper grammar for the readme

### PR DESCRIPTION
**Pull request by NikkiGardiner1**

### CODE REVIEW

File: Exiled.API/Features/Roles/Scp3114Role.cs [hunk 1]
```diff
@@ -204,5 +204,14 @@ public void ResetIdentity()
...
         /// <returns>The Spawn Chance.</returns>
         public float GetSpawnChance(List<RoleTypeId> alreadySpawned) => 0;
+
+        /// <summary>
+        /// Gets or sets the warning time seconds
+        /// </summary>
+        public float WarningTime
+        {
+            get => Identity._warningTimeSeconds;
+            set => Identity._warningTimeSeconds = value;
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp3114Role.cs:214]: Warning is probably not configurable i just need you to verify than it's actually work

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Roles/Scp3114Role.cs (hunk 1 above)
Comment (NikkiGardiner1) [reply Exiled.API/Features/Roles/Scp3114Role.cs:214]: Much like how DisguiseDuration functions. WarningTime also works in a similar manner.

I have tested it and I have verified that it is functional

**Comment by NikkiGardiner1** [comment]
I have updated the README.

There were grammatical errors, such as using it's (it is) and not using its (possessive). Adding 's where needed, lowercasing words that do not need capitalization, removing redundancies, and fixing spelling mistakes.

**Comment by louis1706** [comment]
> I have updated the README.
> 
> There were grammatical errors, such as using it's (it is) and not using its (possessive). Adding 's where needed, lowercasing words that do not need capitalization, removing redundancies, and fixing spelling mistakes.
> 
> The reason it's been added here and not a new PR request is that when making a new PR, GitHub redirected me here since it is still open

The readme part should target Main branch

**Review by louis1706 — APPROVED** [review]
Approved for WarningTime

**Comment by NikkiGardiner1** [comment]
> > I have updated the README.
> > There were grammatical errors, such as using it's (it is) and not using its (possessive). Adding 's where needed, lowercasing words that do not need capitalization, removing redundancies, and fixing spelling mistakes.
> > The reason it's been added here and not a new PR request is that when making a new PR, GitHub redirected me here since it is still open
> 
> The readme part should target Main branch

Ill update it to target the main branch.

**Comment by Misaka-ZeroTwo** [comment]
Could you reverse your readme changes?

**Comment by NikkiGardiner1** [comment]
> Could you reverse your readme changes?

Reversion completed

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**